custom_components/elasticsearch/__init__.py

```diff
@@ -1,279 +1,192 @@
-"""Support for sending event data to an Elasticsearch cluster."""

```

Comment:
🫠 the aggressive formatting changes throughout this PR make reviewing very difficult